data/remnant/remnant outfits.txt

```diff
@@ -474,6 +474,19 @@ outfit "Emergency Ramscoop"
 	"ramscoop" 0.5
 	description "The Ember Waste is the sort of place where only a very foolish captain would travel without some sort of device for replenishing hyperspace fuel. While this risk is non-existent in Remnant ships due to their 'gills', which are effectively built-in ramscoops, some captains prefer to speed up the process by adding additional capacity. For the rare captain braving the area in non-Remnant ships, this ramscoop is not particularly powerful, but it allows a ship to explore without fear of getting stranded."
 
+outfit "NDR-114 Android"
+	category "Systems"
+	licenses
+		Remnant
```

Comment:
Because the Remnant won't just hand them off to someone without a license, and neither will they facilitate installing them to someone without a license either.

Reply:
Except they're gifts, not sold. I would expect a license to be on almost anything the Remnant sell, but not gifts are a different issue.

Reply:
Come to think of it... is it even possible to get this mission before obtaining the basic Remnant license?

If so (and the license requirement remained) would you just not get the androids after completing it?

That would make sense lore wise, but probably isn't the best for gameplay.

And if you can't even get the mission without the license, there's probably not much reason to have a check for a thing the player has anyway.

Reply:
Gifting outfits skips the license check.

Reply:
> Come to think of it... is it even possible to get this mission before obtaining the basic Remnant license?
> 
> If so (and the license requirement remained) would you just not get the androids after completing it?
> 
> That would make sense lore wise, but probably isn't the best for gameplay.
> 
> And if you can't even get the mission without the license, there's probably not much reason to have a check for a thing the player has anyway.

It will be possible to *lose* Remnant licenses, though, which could prevent these from being useful. That being said, does the license check only happen on purchase, or does it apply on install too?

Reply:
Installing an outfit from storage skips the license check. Also, if you sell an outfit that requires a license you don't have, you'll still be able to buy it back and get it installed.

Reply:
Hmmm... Makes sense for some things; don't think it makes sense for *all* things.

Reply:
Well, if it makes no difference either way, then I'm inclined to leave the requirement there for the aesthetic value of matching everything else.

Reply:
Except the android isn't sold so there's not really anything to match.

Reply:
No but it *does* show up in the outfitter when they have them.

Reply:
It shows up when the *player* has them, but the player can go anywhere with them. Why have them match other outfits in Remnant outfitters if there's nothing particular to the androids with relation to Remnant outfits?

Reply:
Anyway, if you *really* don't like it I can remove that. 

This feels like a "I don't think it matters so let's have it this way." vs "You think it doesn't matter so let's have it that way." situation. No sense us wasting a lot of time and effort on something that doesn't really matter.